src/App.tsx

```diff
@@ -1,39 +1,57 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './clock';
```

Comment:
where is clock? 
![image](https://user-images.githubusercontent.com/60798246/190428731-f16f11be-9e1a-4bd2-9dd7-2f5403d30bcb.png)

Reply:
was some problem with deploy, now it's ok